src/core/gltf2exporter/gltf2exporter_p.cpp

```diff
@@ -664,6 +664,19 @@ void GLTF2Exporter::setContextFromImporter(GLTF2Importer *importer)
     m_context = importer->m_context;
 }
 
+QStringList GLTF2Exporter::overwritableFiles(const QDir &target) const
+{
+    QStringList files;
+    for (auto file : m_context->localFiles()) {
```

Comment:
won't that cause a container detach since we aren't doing const auto (or const QString &) ?

Reply:
fixed